src/frontend/komponenter/Oppgavebenk/OppgavelisteNavigator.tsx

```diff
@@ -1,11 +1,28 @@
 import React from 'react';
 
+import styled from 'styled-components';
+
 import { Pagination } from '@navikt/ds-react';
 import { RessursStatus } from '@navikt/familie-typer';
 
 import { oppgaveSideLimit, useOppgaver } from '../../context/OppgaverContext';
 import type { IOppgave } from '../../typer/oppgave';
 
+const StyledDiv = styled.div`
+    color: var(--a-gray-800);
```

Comment:
Jeg synes at disse fargevariablene bør hentes fra ds-tokens for å sikre at de oppdateres i takt med ds. De har allerede byttet navn på dem én gang tidligere, og det er mye lettere å oppdage når det er en variabel som ikke lenger finnes i pakken vi importerer fra enn når det er en hardkodet css-variabel 😇

Reply:
Enig i det! Fikser 🚀